app/models/user.rb

```diff
@@ -18,7 +18,7 @@ class User < Common::RedisStore
   attribute :middle_name
   attribute :last_name
   attribute :gender
-  attribute :birth_date, Common::UTCTime
+  attribute :birth_date
```

Comment:
I've coded this up, and having done so, I've changed my mind :-).  

MVI is the only component that needs knowledge of this "_special formatting_".  Putting this formatted birth_date in the User model pollutes an already overpopulated model.